### PR TITLE
[Feature] controlnet animation inferencer

### PR DESCRIPTION
**Pull request by liuwenran**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

```python
from mmedit.edit import MMEdit

# Create a MMEdit instance and infer
editor = MMEdit(model_name='controlnet_animation')

prompt = 'a girl, black hair, T-shirt, smoking, best quality, extremely detailed'
negative_prompt = 'longbody, lowres, bad anatomy, bad hands, missing fingers, ' + \
                  'extra digit, fewer digits, cropped, worst quality, low quality'

# you can download the example video with this link
# https://user-images.githubusercontent.com/12782558/227418400-80ad9123-7f8e-4c1a-8e19-0892ebad2a4f.mp4
video = '/path/to/your/input/video.mp4'
save_path = '/path/to/your/output/video.mp4'

# Do the inference to get result
editor.infer(video=video, prompt=prompt, negative_prompt=negative_prompt, save_path=save_path)
```

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmediting/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmediting/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: mmedit/models/diffusion_schedulers/__init__.py [hunk 1]
```diff
@@ -76,6 +84,8 @@ def __repr__(self):
         prefix += f'Wrapped Scheduler Name: {self._scheduler_name}\n'
         if self._from_pretrained:
             prefix += f'From Pretrained: {self._from_pretrained}\n'
+        if self._from_config:
+            prefix += f'From Pretrained: {self._from_config}\n'
```
Comment (LeoXing1996) [inline mmedit/models/diffusion_schedulers/__init__.py:88]:
```suggestion
            prefix += f'From Config: {self._from_config}\n'
```

File: configs/controlnet_animation/anythingv3_config.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# config for model
+stable_diffusion_v15_url = 'Linaqruf/anything-v3.0'
+controlnet_hed_url = 'lllyasviel/sd-controlnet-hed'
+control_detector = 'lllyasviel/ControlNet'
+control_scheduler = 'UniPCMultistepScheduler'
```
Comment (LeoXing1996) [inline configs/controlnet_animation/anythingv3_config.py:5]: This variable seems not to be used. 👀

File: configs/controlnet_animation/README.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+|                    Model                    | Dataset |                                     Download                                      |
+| :-----------------------------------------: | :-----: | :-------------------------------------------------------------------------------: |
+| [anythingv3 config](./anythingv3_config.py) |    -    | [stable diffusion model](https://huggingface.co/Linaqruf/anything-v3.0/tree/main) |
+
+## Quick Start
+
+There are two ways to try controlnet animation.
+
+### 1. Use MMedit inference API.
```
Comment (LeoXing1996) [inline configs/controlnet_animation/README.md:55]:
```suggestion
### 1. Use MMEdit inference API.
```

File: configs/controlnet_animation/anythingv3_config.py (hunk 2 above)
Comment (liuwenran) [reply configs/controlnet_animation/anythingv3_config.py:5]: use this variable to change inference scheduler with 'from_config'

File: mmedit/models/diffusion_schedulers/__init__.py (hunk 1 above)
Comment (liuwenran) [reply mmedit/models/diffusion_schedulers/__init__.py:88]: sorry, it is a typo.

**Pull request merged**